virtool/uploads/api.py

```diff
@@ -30,6 +30,8 @@ class UploadsView(PydanticView):
     async def get(
         self,
         user: Optional[str] = None,
+        page: conint(gt=0) = 1,
+        per_page: conint(ge=1, le=100) = 25,
```

Comment:
A bit more consistent.
```suggestion
        page: conint(ge=1) = 1,
        per_page: conint(ge=1, le=100) = 25,
```